sections/baseline_expenses.md

```diff
@@ -0,0 +1,159 @@
+# Baseline Expenses
+
+## Get all baseline expenses in a project
+
+- `GET /projects/{projectId}/baseline_expenses` - Returns all baseline expenses for a given project.
+
+| Response fields         | Description/format                         |
+| ----------------------- | ------------------------------------------ |
+| id                      | Integer                                    |
+| project                 | Integer, ID of project                     |
+| milestone               | Integer, ID of milestone                   |
+| expense_category        | Integer, ID of expense category            |
```

Comment:
We also need the expense categories in the API to have the ID here.
Maybe also update the expenses if there is any changes in the system?

Reply:
Added expense categories in latest revision